webui/src/Page/Survey.elm

```diff
@@ -1025,7 +1025,8 @@ viewSurveyBox answer =
     div [ class "col-md-6" ]
         [ div [ class "card mb-4 box-shadow" ]
             [ div [ class "card-body" ]
-                [ p [ class "card-text h5 mb-4" ] [ text answer.answer ]
+                [ div [ class "card-padder", style [ ( "height", "120px" ) ] ]
```

Comment:
This absolutely does work in browser but I think using the flexbox utilities in bootstrap will handle this more generically and preserve the responsiveness (avoiding hard coded pixel heights)
https://getbootstrap.com/docs/4.1/utilities/flex/#align-items